content/2022/06-30-improvements-in-web-app-integrations-documentation.md

```diff
@@ -0,0 +1,29 @@
+---
+applied_at: "2021-06-30"
+applies_to: 
+- other
+is_impactful: true
+is_new_feature: false
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Improvements in web app integrations documentation
+
+The [Create Web App Integration](1) with a Custom App document is updated to reflect all the information from the developer console.
```

Comment:
```suggestion
The [Create Web App Integration][1] with a Custom App document is updated to reflect all the information in the developer console.
```
The link ref must be in square brackets, otherwise it won't work :( 
I was also thinking about the first part --> `The [Create Web App Integration][1] with a Custom App document` - because now it looks a bit like `Create Web App Integration with a Custom App` is the title. 
Maybe something like `We updated documentation on creating [Web App Integrations][1] with Custom Apps ` or similar?